### PR TITLE
Add EI Systemtests

### PR DESCRIPTION
**Pull request by e-edling-ericsson**
### Description of the Change

This is a merge from the branch systemstests that includes the new systemtests.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: 
Erik Edling erik.edling@ericsson.com

### CODE REVIEW

File: src/systemtest/resources/system-test.properties [hunk 1]
```diff
@@ -0,0 +1,21 @@
+application.name=eiffel-intelligence
+
+# Test Framework logging level
+logging.level: INFO
+
+######## EI Front-End
+ei.frontend.url:http://localhost:8077
+
+######## EI Back-End
+ei.backend.url.artifact:http://localhost:8070
```
Comment (tobiasake) [inline src/systemtest/resources/system-test.properties]: Shouldn't these parameter be using the internal hostnames and inernal ports:
ei-backend-artifact:8080, ei-backend-sourcechanget:8080,,, and so on.

Comment (tobiasake) [reply src/systemtest/resources/system-test.properties]: or $EI_BACKEND_ARTIFACT_HOSTNAME:$EI_BACKEND_ARTIFACT_INTERNAL_PORT

File: src/systemtest/resources/system-test.properties [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ei.backend.url.sourcechange:http://localhost:8072
+ei.backend.url.testexecution:http://localhost:8074
+
+######## Remrem-Publish
+remrem.publish.url:http://localhost:8096
+
```
Comment (tobiasake) [inline src/systemtest/resources/system-test.properties]: Same comment here for RemRem,,, internal port and internal hostname.

File: src/systemtest/java/com/ericsson/ei/systemtest/utils/Config.java [hunk 3]
```diff
@@ -37,11 +38,13 @@ public Config() {
     * @return
     */
     public void initJenkinsConfig() {
-        jenkinsBaseUrl = System.getProperty("jenkins.url");
+        jenkinsExternalBaseUrl = System.getProperty("jenkins.external.url");
```
Comment (e-pettersson-ericsson) [inline src/systemtest/java/com/ericsson/ei/systemtest/utils/Config.java]: Maybe add a comment to these variables for future references? To explain when the external vs the internal URL are used.

**Pull request merged**